zephyr/samples/someip_sd_client/src/main.cpp

```
@@ -0,0 +1,132 @@
+using namespace someip::sd;
+using namespace someip::transport;
+
+static constexpr uint16_t SERVICE_ID   = 0x1000;
+static constexpr uint16_t METHOD_HELLO = 0x0001;
+
+static std::atomic<bool> discovered{false};
+static std::string disc_address;
+static uint16_t    disc_port = 0;
+
+static void on_found(const std::vector<ServiceInstance>& services) {
+    for (auto& svc : services) {
```

Cppcheck (2.19.0)

[information] Include file

(missingIncludeSystem)